src/RecoveryServices/RecoveryServices/help/New-AzRecoveryServicesVault.md

```diff
@@ -102,6 +102,7 @@ Position: Named
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
+Locked state: True
```

Comment:
These help files are generated using PlatyPS. These field will be overwrote next time we refresh it. Please add these into the help message in `.cs` file and regenerate the help file.